frontend/src/pages/TermPlanner/TermPlanner.jsx

```diff
@@ -170,7 +170,7 @@ const TermPlanner = () => {
               <div className="gridItem">Term 1</div>
               <div className="gridItem">Term 2</div>
               <div className="gridItem">Term 3</div>
-              <div className="gridItem">Unplanned</div>
+              <div className="gridItem" />
```

Comment:
Maybe add a comment saying that this empty div is for adding the spacing for the fix unplanned column?